lib/admin_web/forms/published_item_search_form.ex

```diff
@@ -35,14 +37,33 @@ defmodule AdminWeb.Forms.PublishedItemSearchForm do
   defp maybe_validate_id_exists(changeset) do
     item_id = Ecto.Changeset.get_field(changeset, :item_id)
 
-    if Publications.exists?(item_id) do
+    if Publications.item_exists?(item_id) do
       changeset
     else
       Ecto.Changeset.add_error(
         changeset,
         :item_id,
-        "Publication with id '#{item_id}' could not be found"
+        "An Item with id '#{item_id}' could not be found"
       )
     end
   end
+
+  @spec maybe_put_published_item_id(Ecto.Changeset.t()) :: Ecto.Changeset.t()
+  defp maybe_put_published_item_id(%Ecto.Changeset{valid?: false} = changeset), do: changeset
+
+  defp maybe_put_published_item_id(changeset) do
+    item_id = Ecto.Changeset.get_field(changeset, :item_id)
+
+    case Publications.get_published_item_id_for_item_id(item_id) do
+      nil ->
+        Ecto.Changeset.add_error(
+          changeset,
+          :published_item_id,
+          "Publication does not exist for item with id '#{item_id}'"
+        )
+
+      published_item_id ->
+        Ecto.Changeset.put_change(changeset, :published_item_id, published_item_id)
+    end
+  end
```

Comment:
I think this whole "module"(?) is a bit confusing because isn't `item_id` supposed to be the same as `published_item_id`? I think it's not clear `item_id` is actually a query from a search. Maybe rename `item_id`.
Also wouldn't it be quicker to check for `item_id` within the context of "published item" instead of first checking whether the `item_id` is an existing item, but might not be a published item?